src/services/balancer/contracts/contracts/batch-relayer.ts

```diff
@@ -0,0 +1,78 @@
+import { Contract } from 'ethers';
+import ContractService from '../balancer-contracts.service';
+import BatchRelayerAbi from '@/lib/abi/BatchRelayer.json';
+import { FundManagement, TransactionData } from '@balancer-labs/sdk';
+import { TransactionResponse, Web3Provider } from '@ethersproject/providers';
+import { sendTransaction } from '@/lib/utils/balancer/web3';
+
+export default class BatchRelayer {
+  service: ContractService;
+  instance: Contract;
+
+  constructor(service, public readonly abi = BatchRelayerAbi) {
+    this.service = service;
+    this.instance = new Contract(
+      this.service.config.addresses.batchRelayer,
+      this.abi,
+      this.service.provider
+    );
+  }
+
+  public get address(): string {
+    return this.service.config.addresses.batchRelayer;
+  }
+
+  public async stableExitStatic(
+    account: string,
+    tokenIn: string,
+    amountsIn: string[],
+    tokensOut: string[],
+    rates: string[],
+    slippage: string,
+    exactOut = false
+  ): Promise<TransactionData> {
+    const funds: FundManagement = {
+      sender: account,
+      recipient: this.address, // Note relayer is recipient of swaps
+      fromInternalBalance: false,
+      toInternalBalance: false
+    };
+
+    const tokensIn = tokensOut.map(() => tokenIn);
+
+    const method = exactOut
+      ? 'swapUnwrapAaveStaticExactOut'
+      : 'swapUnwrapAaveStaticExactIn';
+
+    console.log([
+      method,
+      tokensIn,
+      tokensOut,
+      amountsIn,
+      rates,
+      funds,
+      slippage
+    ]);
```

Comment:
console.log needed?

Reply:
No was for debugging yesterday, will remove.